alf/algorithms/muzero_algorithm.py

```diff
@@ -109,13 +112,24 @@ def __init__(self,
                 normalize reward.
             train_reward_function (bool): whether train reward function. If
                 False, reward should only be given at the last step of an episode.
-            calculate_priority (bool): whether to calculate priority. This is
-                only useful if priority replay is enabled.
+            calculate_priority (bool): whether to calculate priority. If not provided,
+                will be same as ``TrainerConfig.priority_replay``. This is only
+                useful if priority replay is enabled.
             train_game_over_function (bool): whether train game over function.
+            reanalyze_mcts_algorithm_ctor (Callable): will be called as
+                ``mcts_algorithm_ctor(observation_spec=?, action_spec=?, debug_summaries=?, name=?)``
```

Comment:
We can probably do the following minor change, although they are the same by default:
mcts_algorithm_ctor(...) -> reanalyze_mcts_algorithm_ctor(...)